conf/machine/imx8qm-mek.conf

```diff
@@ -42,7 +42,6 @@ KERNEL_DEVICETREE = " \
     freescale/imx8qm-mek-hdmi-rx-ov5640.dtb \
     freescale/imx8qm-mek-jdi-wuxga-lvds1-panel.dtb \
     freescale/imx8qm-mek-jdi-wuxga-lvds1-panel-rpmsg.dtb \
-    freescale/imx8qm-mek-ov5640.dtb \
```

Comment:
The changes to `KERNEL_DEVICETREE` for these machines are in `meta-imx/meta-imx-bsp/conf/layer.conf`.

Reply:
Hi @thochstein . I have updated the device-tree list on the imx8qm-mek. However, the meta-imx has a configuration for IMXBOOT_TARGETS_SD adding flash_linux_m4 but it fails because of imx-boot missing changes.
@otavio asked if you could look at this?

Reply:
Thanks @mdrodrigo. I'm looking at it now.

Reply:
I'm not seeing any build break as-is. I don't see anything in particular in this PR that would cause it. Can you reproduce the problem without this PR? I need some help reproducing your issue.

Reply:
Hi @thochstein . #2238